FastMember/TypeAccessor.cs

```diff
@@ -34,6 +35,11 @@ public abstract class TypeAccessor
         /// </summary>
         public virtual MemberSet GetMembers() { throw new NotSupportedException(); }
 
+        /// <summary>
+        /// Query the reflected members discovered for this type and scope
+        /// </summary>
+        public virtual MemberInfo[] CachedMembers { get; private set; }

```

Comment:
Many people, including me, have asked for cached reflection so we don't have to do our own alongside use of FastMember. This is a side effect of wanting to keep these around for `MemberSet`.